app/src/main/java/org/p2p/wallet/moonpay/repository/MoonpayBuyRepository.kt

```diff
@@ -5,7 +5,7 @@ import org.p2p.wallet.moonpay.api.MoonpayIpAddressResponse
 import org.p2p.wallet.moonpay.model.MoonpayBuyResult
 import java.math.BigDecimal
 
-interface MoonpayRepository {
+interface MoonpayBuyRepository {
```

Comment:
each moonpay features come as separate repository to split responsibility
so we have 
MoonpaySell
MoonpayBuy